src/models/CommentsModel.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+require_once 'src/entities/CommentEntity.php';
+
+class CommentsModel
+{
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function addComment(CommentEntity $commentEntity): bool
+    {
+        $query = $this->db->prepare(
+            'INSERT INTO `comments` (`content`, `username_id`, `post_id`, `date_posted`, `time_posted`) VALUES (:content, :username_id, :post_id, :date_posted, :time_posted);');
+        return $query->execute([
+            ':content' => $commentEntity->content,
+            ':username_id' => $commentEntity->username_id,
+            ':post_id' => $commentEntity->post_id,
+            ':date_posted' => $commentEntity->date_posted,
+            ':time_posted' => $commentEntity->time_posted]);
+    }
+    static public function AddCommentToDatabase(int $id, string $content): CommentEntity
```

Comment:
This doesn't add a comment to the database, it creates a comment entity. Actively misleading function name!

You could probably use the constructor of a comment entity to do this whole thing.

Either way it's not saving much work over either:
- instantiate the new comment entity in the one place in your code you want to do that and fill in its fields just like this
- pass in the `post_id`, `title` and `content` of a comment into `addComment`, skipping creating an entity entirely.